requirements.txt

```diff
@@ -1,6 +1,6 @@
 matplotlib
 scipy
-numpy
+numpy>=1.9.5
```

Comment:
you mean 1.19.5 I assume, 1.9.5 is quite old

Reply:
ho yes of course!